scripts/train_keras.py

```diff
@@ -439,6 +491,6 @@ def evaluate(model_root, model, images, image_size, num_batches, seed):
     if FLAGS.score:
         logger.info('Model and description saved. Evaluating and scoring.')
         classes, cm_path, metrics = evaluate(model_root, trained_model, FLAGS.image_dir, im_sz,
-                 FLAGS.num_batches_to_score, FLAGS.seed)
-    write_model_desc(FLAGS, FLAGS.model_dir, model_name, classes, weights,
-                     training_data, cm_path, metrics)
+            FLAGS.num_batches_to_score, FLAGS.seed)
+        write_model_desc(FLAGS, FLAGS.model_dir, model_name, classes, weights,
```

Comment:
Actually, we _always_ want to write_model_desc, even if we're not scoring. If we're not scoring, that part of the markdown is ignored.